deb/openmediavault/srv/salt/omv/deploy/apt/files/etc-apt-sources_list_d-openmediavault_list.j2

```diff
@@ -1,19 +1,17 @@
-{% set repo_url = salt['pillar.get']('default:OMV_APT_REPOSITORY_URL', 'http://packages.openmediavault.org/public/') -%}
-deb {{ repo_url }} {{ pillar['productinfo']['distribution'] }} main
-deb https://openmediavault.github.io/packages/ {{ pillar['productinfo']['distribution'] }} main
-# deb http://downloads.sourceforge.net/project/openmediavault/packages/ {{ pillar['productinfo']['distribution'] }} main
+{% set public_repo_url = salt['pillar.get']('default:OMV_APT_REPOSITORY_URL', 'http://packages.openmediavault.org/public/') -%}
+{% set packages_repo_url = salt['pillar.get']('default:OMV_PACKAGES_REPOSITORY_URL', 'https://openmediavault.github.io/packages/') -%}
```

Comment:
Using a variable here makes no sense, the URL will never change, except you're rebranding OMV and ship it under your own url. Please remove.

Reply:
plz re-review.

Reply:
Now he is mirrored, https://mirrors.tuna.tsinghua.edu.cn/OpenMediaVault/packages